src/scripts/modules/media/media.coffee

```diff
@@ -23,26 +23,27 @@ define (require) ->
 
   ###
   Returns an event handler that will handle hiding and showing
-  items based on the scroll direction. Attach the handler to
+  items based on the scroll position. Attach the handler to
   a scroll event to hook it up.
   ###
-  headerController = ($el, selector) ->
-    oldTop = 0
-    inTransition = false
+  headerController = ($el, selector, compactSelector) ->
+    threshold = window.innerHeight / 5
+    fudge = 8
+    hideAbove = threshold
+    showBelow = fudge
```

Comment:
What's the point of the `fudge` variable if it's just being set to `showBelow` and then never used again?